apps/minaintyg/src/components/Layout/GlobalBanner/GlobalBanner.tsx

```diff
@@ -0,0 +1,13 @@
+import { IDSAlertGlobal, IDSIconAttention, IDSIconInformation, IDSIconWarning } from '@frontend/ids-react-ts'
+import { Banner, BannerPriority } from '../../../schema/info.schema'
+
+export function GlobalBanner({ banner }: { banner: Banner }) {
```

Comment:
Shouldn't this be a common component between apps and not placed in MI? Maybe there's code from RS that can be reused?

Reply:
There is a difference in BannerPriority that led me to implement a new Component.